src/lib/helpers/isEditableHtmlTextPopupVisible.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Checks if Link Popup of editable html text is visibile
+ * @returns
+ */
+export const isEditableHtmlTextPopupVisible = () => {
+  return (
+    document.querySelector(".mantine-RichTextEditor-linkEditorDropdown") !==
```

Comment:
Was able to quick-fix this 👍 

In `RichTextarea.tsx` :

```typescript
export const RichTextarea: React.FC<
  Omit<RichTextEditorProps, "editor" | "onChange" | "children"> & {
    label?: string;
    value: string;
    onChange: React.Dispatch<React.SetStateAction<string>>;
}>
```

make it possible to pass in `Editor` props (just remove those which conflicts with the component's own props.. check if there are more props to omit, but that should do the trick)

so that now we can spread them in the `Editor`

```typescript
({ label, value, onChange, ...rest }) => [...]

...

<RichTextEditor editor={editor} {...rest}>
```

Then in `EditableHtmlText`

```typescript
<RichTextarea
          classNames={{
            linkEditorDropdown: "my-shiny-link-editor-class",
          }}
          value={internalValue}
          onChange={setInternalValue}
        />
```

and then you can query to your specific classname (`my-shiny-link-editor-class` in this example)

Reply:
Applied a fixed class for richTextArea's linkEditorDropdown